docs/content/users/quickstart.md

```diff
@@ -470,7 +510,15 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
 ## Configuration Files
 
-The [`ddev config`](../users/usage/commands.md#config) command attempts to create a CMS-specific settings file pre-populated with DDEV credentials.
+The [`ddev config`](../users/usage/commands.md#config) and `ddev start` commands attempt to create a CMS-specific settings file pre-populated with DDEV credentials. If you don't want DDEV to do this, use `settings_management_disabled: true` in the `.ddev/config.yaml`. `
+
+!!!note "Completely Disabling Settings Management"
+
+    If you do *not* want DDEV to create or manage settings files, set `disable_settings_management: true` in `.ddev/config.yaml` or run `ddev config --disable-settings-management`. Once you’ve done that, it’s solely up to you to manually edit those settings.
```

Comment:
I’d remove this since it was literally the last thing written before the admonition, and there’s a [whole separate page on the topic](https://ddev--4768.org.readthedocs.build/en/4768/users/usage/cms-settings/) that covers it more thoroughly.

```suggestion
```